src/main/java/seedu/duke/atomichabit/command/ListCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.duke.atomichabit.command;
+
+import seedu.duke.atomichabit.atomichabit.AtomicHabit;
+import seedu.duke.atomichabit.atomichabit.AtomicHabitList;
+
+public class ListCommand extends Command{
+
+    private final String LINE_SEPARATOR = System.lineSeparator();
+    private final String DOT = ".";
+    private final String FIRST_STRING = "Here are all the habits in your list:";
+    public static final String COMMAND_WORD = "list";
+    @Override
+    public CommandResult execute(AtomicHabitList atomicHabitList) {
+        int taskNo = 1;
+        String list = FIRST_STRING + LINE_SEPARATOR;
+        for(AtomicHabit ab : atomicHabitList.getAllHabits()) {
```

Comment:
Add space after `for`

Reply:
Resolved in 3371f317f0f77df8dbf6210446555e6af3ebac85.